Library/Formula/libquantum.rb

```diff
@@ -0,0 +1,35 @@
+class Libquantum < Formula
+  desc "C library for the simulation of quantum mechanics"
+  homepage "http://www.libquantum.de/"
+  url "http://www.libquantum.de/files/libquantum-1.0.0.tar.gz"
```

Comment:
1.0.0 is the stable release, while 1.1.1 is the development release.  Refer to http://www.libquantum.de/downloads